packages/wonder-blocks-form/src/components/__tests__/text-field.test.tsx

```diff
@@ -158,24 +158,6 @@ describe("TextField", () => {
         expect(input).toBeInTheDocument();
     });
 
-    it("disabled prop disables the input element", async () => {
```

Comment:
Moved this test with the other `Disabled state` tests! It was also updated to check for the `aria-disabled` attribute instead